### PR TITLE
Create new Profile

### PR DESCRIPTION
**Pull request by GDeppe**

### CODE REVIEW

File: validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "contact": {
+    "email": "f3hd@gmail.com",
+    "telegram": "",
+    "twitter": "",
+    "website": ""
```
Comment (terran6) [inline validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json:6]: @GDeppe You can go ahead and remove these lines if there are no values to add here

**Review by terran6 — CHANGES_REQUESTED** [review]

File: validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json (hunk 1 above)
Comment (GDeppe) [reply validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json]: This has now been completed and resubmitted

File: validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "contact": {
+    "email": "f3hd@gmail.com",
```
Comment (terran6) [inline validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json:3]: @GDeppe Please remove the trailing comma on this line to adhere to JSON formatting.

**Review by terran6 — CHANGES_REQUESTED** [review]

File: validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json (hunk 2 above)
Comment (GDeppe) [reply validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json]: trailing comma removed

**Comment by Mrt3383x38** [comment]
Siz insanlarla dalgamı geciyosunuz nerede aydrop aldığı nızı verin artık yeter

Android için Outlook<https://aka.ms/AAb9ysg> edinin
________________________________

File: validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json (hunk 2 above)
Comment (Mrt3383x38) [reply validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json]: - [ ] **@































 **

**Comment by Mrt3383x38** [comment]
> 

Nerde binance yede giremiyoruz iyi eyleniyonuzmu

**Review by terran6 — APPROVED** [review]

**Pull request merged**